docusaurus/docs/publish-a-plugin/build-automation.md

```diff
@@ -18,42 +18,171 @@ keywords:
 We recommend setting up your plugin to use the supplied [Github workflows](../get-started/set-up-development-environment.mdx#set-up-github-workflows) from [create-plugin](../get-started/get-started.mdx)
 to ensure that your plugin will be built and packaged in the correct format.
 
-To do so, refer to [these docs](https://github.com/grafana/plugin-actions/blob/main/build-plugin/README.md).
-
 Additionally, we recommend using the zip file produced from this workflow to test the plugin.
 
 If a Grafana Access Policy Token is included in your [Github repository secrets](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-development-environment-secrets-for-your-repository-or-organization), a signed build is automatically created, which you can use to test the plugin locally before submission.
 
 By creating a release tag, the whole process becomes automated, resulting in a zip file that you can submit for publication to the [Grafana plugin catalog](https://grafana.com/plugins)
 
-The zip file can be downloaded from the summary page of the CI workflow.
+You can use the links to the archive and zip files from the release page to make your plugin submission.
+
+## Setup the release workflow
+
+Ensure your repository contains a `.github/workflows/release.yml` file with the following contents: 
```

Comment:
```suggestion
By default, create-plugin will add the following `release.yml` to your scaffolded plugin with the following contents. If this is missing from your plugin repository, copy the following to add the workflow: 
```
Nit - but given its the default and we are generally recommending that people use the default supplied workflow, which this is - maybe instead we can remove the note and just tweak this text a bit to clean it up

Reply:
it is the default but there are plenty of plugins that were scaffolded before it was. that's why I am trying to make it clear. 

But I like the idea of tweaking this text better